### ISSUE TITLE
BUG: Enemies automatically open fire on CamSpy and variants in Co-op

### ISSUE THREAD

**Issue by NotFragile74** [fgsfdsfgs/perfect_dark#453]
In Co-op mode, whenever I deploy a CamSpy, DrugSpy or BombSpy, rather than running towards it and looking at it for a few seconds, enemies will instantly open fire on it and destroy it. This glitch was present in the original N64 version and the XBLA remaster.

**Comment by FrostKnight** [comment]
Was this  a glitch? I thought it was on purpose.

**Comment by NotFragile74** [comment]
> Was this a glitch? I thought it was on purpose.

Not like this. See, in single-player campaign, if you deploy the CamSpy - or any other Spy - for that matter, if enemies see it, they will run up to it and observe it curiously for a moment before deducing that it's a threat and shooting at it. The glitch is that in Co-op mode, if you deploy the CamSpy or whatever, if enemies see it, they will instantly shoot it on sight without asking questions.

**Comment by LonelySpaceDetective** [comment]
> See, in single-player campaign, if you deploy the CamSpy - or any other Spy - for that matter, if enemies see it, they will run up to it and observe it curiously for a moment before deducing that it's a threat and shooting at it. The glitch is that in Co-op mode, if you deploy the CamSpy or whatever, if enemies see it, they will instantly shoot it on sight without asking questions.

Yeah, a good way of checking this out is playing Mr. Blonde's Revenge in co-op.
In singleplayer when you use the BombSpy, you can easily corral the bodyguards together to make the most out of the detonation, but in co-op they just immediately fire on it, making for a rather difficult firefight right at the start of the level.
